### PR TITLE
feat: remove nft-devnet

### PR DESCRIPTION
**Pull request by ckniffen**
## High Level Overview of Change

This devnet will be decommissioned soon

### Type of Change

- [x] New feature (non-breaking change which adds functionality)

## After
Monosnap XRPL Explorer | TX F9D17A67  2023-01-24 17-33-32

### CODE REVIEW

**Review by intelliot — APPROVED** [review]
Given we've mentioned the nft-devnet on the blog before, can we post a brief explaining why it is being removed?

e.g. https://xrpl.org/blog/2022/nft-devnet-reset.html

/cc @mDuo13

**Review by shawnxie999 — APPROVED** [review]

**Comment by ckniffen** [comment]
> Given we've mentioned the nft-devnet on the blog before, can we post a brief explaining why it is being removed?
> 
> e.g. https://xrpl.org/blog/2022/nft-devnet-reset.html
> 
> /cc @mDuo13

Would redirecting to the blog post suffice?

File: .env.example [hunk 1]
```diff
@@ -13,7 +13,6 @@ REACT_APP_MAINNET_LINK=
 REACT_APP_TESTNET_LINK=
 REACT_APP_DEVNET_LINK=
 REACT_APP_AMM_LINK=
-REACT_APP_NFTSANDBOX_LINK=
```
Comment (ledhed2222) [inline .env.example:16]: remove reference to this env in comment on line 19/20 as well

**Comment by ledhed2222** [comment]
A few additional places I found references to the sandbox:

`.env.example` line 20
`public/locales/en-US/translations.json` line 11
`.vscode/settings.json` line 68

**Review by ledhed2222 — APPROVED** [review]

**Comment by intelliot** [comment]
> > Given we've mentioned the nft-devnet on the blog before, can we post a brief explaining why it is being removed?
> > e.g. https://xrpl.org/blog/2022/nft-devnet-reset.html
> > /cc @mDuo13
> 
> Would redirecting to the blog post suffice?

I meant: There is no blog post saying that nft-devnet is being shut down or removed. We should create such a blog post, so the community is aware of the change (and why).

**Comment by ckniffen** [comment]
> I meant: There is no blog post saying that nft-devnet is being shut down or removed. We should create such a blog post, so the community is aware of the change (and why).

We now have one.  https://xrpl.org/blog/2023/nft-devnet-decommission.html.

**Pull request merged**